gradle.properties

```diff
@@ -1,3 +1,17 @@
-cdvCompileSdkVersion=32
+## For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+#
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+# Default value: -Xmx1024m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+#
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+#Fri Jan 14 14:58:47 MST 2022
 cdvMinSdkVersion=24
+cdvCompileSdkVersion=32
+org.gradle.jvmargs=-Xmx2048M -Dkotlin.daemon.jvm.options\="-Xmx2048M"
```

Comment:
I think this file was modified by the IDE and I would not be opposed to reverting it if people think it should be.

Reply:
If you think you need those JVM args, then keep them.

Reply:
I think this makes the emulators more responsive, but it's dependent on the limitations of the machine.  I vaguely remember tweaking IDE memory settings toward the beginning of this endeavor, but I feel like this should be in a local config file that isn't committed.  I'm honestly leaning toward reverting this now that I'm combing through each modified file.